assets/xml/overlays/ovl_Arrow_Fire.xml

```diff
@@ -0,0 +1,11 @@
+<Root>
+    <File Name="ovl_Arrow_Fire" BaseAddress="0x80865d10" RangeStart="0x9D0" RangeEnd="0x1DF0" Segment="128">    
+        <Texture Name="sFireArrowTex1" OutName="fire_tex_1" Format="ia8" Width="32" Height="64" Offset="0x9D0"/>
```

Comment:
Should all of these be prefixed with `s` or `g`?

Reply:
While they are currently accessible globally they *should* probably be static later, however ZAPD doesn't have a manual static override for anything manually extracted.